pyrometheus/__init__.py

```diff
@@ -409,7 +435,7 @@ def get_concentrations(self, rho, Y):
         conctest = self.iwts * rho * Y
```

Comment:
Why conctest? My issue with this variable name is the 'test' part. Can we call it concentrations or simply c?